Sources/muterCore/Rewriters/MutationSwitch.swift

```diff
@@ -15,31 +15,26 @@ struct MutationSwitch {
 
         var previousElseBody = IfExprSyntax.ElseBody(
             CodeBlockSyntax(
-                leftBrace: .leftBraceToken()
-                    .withTrailingTrivia(
-                        originalSyntax.trailingTrivia
-                    ),
-                statements: originalSyntax,
+                statements: originalSyntax.appendingLeadingTrivia(.tabs(1)),
```

Comment:
Bold choice to use tabs 😆 but I wonder if there’s a reasonable way to detect the indentation of the surrounding code, or the indentation setting in the Xcode project? Or is tabs the least-worst choice here? Either way, it would be good to add a comment here, explaining the decision.

Reply:
Hi @ZevEisenberg, I've added a comment

Reply:
Nice! In future, we could have a flag to override it if we needed, but let's see how it works in practice.

Reply:
@ZevEisenberg  I just realized 1 thing when I tried to update the unit tests.

The problem is Xcode always treats the indentation at the start of the line as spaces.
It means if we have a piece of code as text, then we ask Xcode to generate a Syntax -> The Syntax always use spaces.
Meanwhile, since I add `tabs(1)` -> the syntax that is generated by `MuterRewriter` will always use tabs.

It leads to the AssertSnapshot always fail even though both `syntax.description` look identical 

![Screenshot 2024-01-03 at 11 49 26 PM](https://github.com/muter-mutation-testing/muter/assets/48347505/0e01498f-32ed-4ae9-a447-0db84d2233af)

Reply:
So my idea is instead of adding `tabs(1)`, I will add `spaces(x)`. the `x` value will be retrieved from the `config.yml` file, with the default value is 4.

Wdyt?